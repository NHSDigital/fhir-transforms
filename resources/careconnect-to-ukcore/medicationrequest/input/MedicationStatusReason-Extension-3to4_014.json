{"resourceType":"MedicationRequest","extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatusReason-1","extension":[{"url":"statusReason","valueCodeableConcept":{"text":"Recreate 22047"}},{"url":"statusChangeDate","valueDateTime":"2011-11-15T14:52:00.197+00:00"}]}]}
