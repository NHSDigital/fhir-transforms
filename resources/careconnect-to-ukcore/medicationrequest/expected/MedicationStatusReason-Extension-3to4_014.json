{
  "resourceType": "MedicationRequest",
  "extension": [
    {
      "extension": [
        {
          "url": "statusChangeDate",
          "valueDateTime": "2011-11-15T14:52:00.197+00:00"
        }
      ],
      "url": "http://fhir.nhs.uk/fhir/3.0/StructureDefinition/extension-statusChangeDate"
    }
  ],
  "statusReason": {
    "text": "Recreate 22047"
  }
}